marshaling.go

```diff
@@ -131,31 +131,94 @@ type WorksheetConverter interface {
 
 var worksheetConverterType = reflect.TypeOf((*WorksheetConverter)(nil)).Elem()
 
-func (ws *Worksheet) StructScan(dest interface{}) error {
+// wsDestination is used during structScan to properly populate reused non-pointer struct references
+type wsDestination struct {
+	dest interface{}
+	loci []reflect.Value
+}
+
+func (wsd *wsDestination) setAll() {
+	for _, locus := range wsd.loci {
+		destPtr := reflect.ValueOf(wsd.dest)
+		// dests are stored as pointers but we are setting non-pointer destinations
+		locus.Set(destPtr.Elem())
+	}
+}
+
+type wsDestinationMap map[string]*wsDestination
+
+func (wsdm wsDestinationMap) setAllDestinations() {
+	for _, d := range wsdm {
+		d.setAll()
+	}
+}
+
+func (wsdm wsDestinationMap) newDestination(destId string, dest interface{}) {
```

Comment:
rename to `addDestination`?

and signature could be `addDestination(ws *Worksheet, dest interface{})`